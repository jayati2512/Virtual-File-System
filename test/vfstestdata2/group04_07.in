createvfs demo 1024 
createvfs demo 1024
mountvfs demo
umountvfs demo
makedir / files
mountvfs demo
makedir / files
makedir /download/ folder1
deletedir /download/folder1
addfile /download/ file1.txt /usr/include/
addfile /files/ file3.txt /usr/include/
removefile /download/ file3.txt
umountvfs demo
addfile /download/ file0.txt /usr/include/
mountvfs demo
addfile /download/ file2.txt /usr/include/
updatefile /download/file2.txt /usr/include/list2.txt
movefile /download/file2.txt /files/file3.txt
exportfile /download/file2.txt /usr/include/
searchfile file0 output.txt
unmountvfs demo
