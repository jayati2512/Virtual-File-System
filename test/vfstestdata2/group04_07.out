createvfs_SUCCESS
createvfs_FAILURE DATA_FILE_ALREADY_EXISTS
mountvfs_SUCCESS
unmountvfs_SUCCESS
makedir_FAILURE VFS_NOT_MOUNTED
mountvfs_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
deletedir_FAILURE DIRECTORY_NOT_EMPTY
addfile_SUCCESS
addfile_SUCCESS
removefile_SUCCESS
unmountvfs_SUCCESS
addfile_FAILURE VFS_NOT_MOUNTED
mountvfs_SUCCESS
addfile_SUCCESS
updatefile_SUCCESS
movefile_SUCCESS
exportfile_FAILURE VFS_INSUFFICIENT_ARGUMENT 
searchfile_FAILURE CANNOT_FIND_SOURCE_FILE
unmountvfs_SUCCESS
